src/components/pages/career-details/career-details-structured-data.tsx

```diff
@@ -27,12 +21,9 @@ const normalizeJobScheduleFormat = (schedule: string): string => {
  * Reference:
  * https://developers.google.com/search/docs/advanced/structured-data/job-posting#rdescription
  */
-const generateHtmlDescription = (position: SyPersonioJob): string => {
-  const htmlIntro = `${position.short} <br><br>`;
-  const htmlSections = position.sections
-    .map((section) => `${section.headline} <br> ${section.descriptionHtml}`)
-    .join('<br><br>');
-  return htmlIntro + htmlSections;
+const generateHtmlDescription = (position: ContentfulVacancy): string => {
+  const richText = renderRichText(position.content, {});
+  return renderToString(<>{richText}</>);
```

Comment:
According to the [Google Guide](https://developers.google.com/search/docs/appearance/structured-data/job-posting#job-posting-definition) for job descriptions, only certain html tags are allowed. Other tags are apparently simply ignored ("_While the tags won't affect the formatting in the feature, you can safely include them on the page._"). Before the changes, there were only supported tags in the description. I'm not quite sure if it makes sense to parse the description exactly as before or leave it as it is now (with unsupported tags like h1). 
All other fields that are relevant for Google didn't change (tested with https://search.google.com/test/rich-results)


<details>
  <summary>Google Job Description (Before)</summary>
<img width="843" alt="Bildschirm­foto 2023-03-14 um 08 45 06" src="https://user-images.githubusercontent.com/57712895/224931187-6c8e41b6-4fa5-402f-a268-4d1164180bc6.png">
</details>

<details>
  <summary>Google Job Description (Updated)</summary>
<img width="845" alt="Bildschirm­foto 2023-03-14 um 09 04 16" src="https://user-images.githubusercontent.com/57712895/224935019-1c18e913-db9f-43e8-abc9-6db5dd71a5ae.png">


</details>